ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/vlan-interfaces.j2

```diff
@@ -201,6 +201,19 @@ interface {{ vlan_interface }}
 {%             if vrid.ipv6.address is arista.avd.defined %}
    vrrp {{ vrid.id }} ipv6 {{ vrid.ipv6.address }}
 {%             endif %}
+{%             if vrid.tracked_object is arista.avd.defined %}
```

Comment:
The next line, you used arista.avd.natural_sort filter, which actually does a defined test. So we can remove this redundant test in this case.